File: Logic_HandsOn/Java/Problems on bit,file handling,String etc/Program103.java
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Write a program which accept string from user and toggle the case.
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Solution
{
	String str;
	int iCnt;

	Solution(String str)
	{
		this.str=str;
		this.iCnt=0;
	}

	void strtogglex()
	{
		
		while(iCnt!=str.length())
		{
			if(str.charAt(iCnt)>='A' && str.charAt(iCnt)<='Z')
			{
				System.out.print((char)(str.charAt(iCnt)+32));
			}
			else if(str.charAt(iCnt)>='a' && str.charAt(iCnt)<='z')
			{
				System.out.print((char)(str.charAt(iCnt)-32));
			}
			else
			{
				System.out.print(str.charAt(iCnt));
			}
			iCnt++;
		}
		System.out.println();
	}
}

class Program
{
	public static void main(String args[])
	{
		Scanner sObj = new Scanner(System.in);
		System.out.println("Enter the string: ");
		String str = sObj.nextLine();

		Solution obj = new Solution(str);
		obj.strtogglex();
	}
}

/*
OUTPUT

Enter the string:
Vishal PAtil
vISHAL paTIL

*/

File: Logic_HandsOn/C++/Problems on bit,File Handing,Pattern etc/program103.cpp
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Write a program which accept string from user and toggle the case.
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////



#include<iostream>
using namespace std;

class Solution
{

	private:
		string str;
		int iCnt,iCount;

	public:

		Solution(string str,int iCnt=0,int iCount=0)
		{
			this->str=str;
		}

		void strtogglex()
		{
			while(iCnt!=str.length())
			{
			
				if(str[iCnt]>='A' && str[iCnt]<='Z')
				{
					cout<<(char)(str[iCnt]+32);
				}
				else if(str[iCnt]>='a' && str[iCnt]<='z')
				{
					cout<<char(str[iCnt]-32);
				}
				else
				{
					cout<<str[iCnt];
				}
			
				iCnt++;	
			}	
			cout<<"\n";
		}
};

int main()
{
	string str;

	cout<<"Enter string: ";
	getline(cin,str);
	
	Solution *sObj = new Solution(str);
	sObj->strtogglex();

	delete sObj;
	return 0;

}

/*
OUTPUT:
Enter string: Marvellous Multi OS
mARVELLOUS mULTI os


*/
